mappings/net/minecraft/entity/ai/goal/FollowTargetGoal.mapping

```diff
@@ -1,4 +1,21 @@
 CLASS aog net/minecraft/entity/ai/goal/FollowTargetGoal
+	FIELD a targetEntityClass Ljava/lang/Class;
 	FIELD b reciprocalChance I
+	FIELD c targetEntity Laii;
```

Comment:
`target`

Reply:
This conflicts with `target` in `TrackTargetGoal` if `TrackTargetGoal` is also changed

Reply:
How does it conflict? It just hides the field in the super class, I don't see a problem.

Reply:
I'm seeing `Name is not unique in net/minecraft/entity/ai/goal/TrackTargetGoal!` in enigma when I try changing this